tools/footdown.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+
+import os
+import tensorflow as tf
```

Comment:
Wrap potentially contentious imports in a try...except

Reply:
Should do this for all of them and provide/not provide a b command depending on this (another PR)